sippy-ng/src/prow_job_runs/ProwJobRun.js

```diff
@@ -953,7 +1011,10 @@ function createTimelineData(
       startDate = earliest
     }
     let endDate = new Date(item.to)
-    if (!item.to) {
+
+    // When go unmarshalls a null "to", the date is 0001-01-01T00:00:00Z which
+    // we're assuming is null.
```

Comment:
This comment isn't accurate, a null value will be marshalled as null.  It only marshalls the zero value (0001-01-01) when using a non-pointer type.  Go code should probably be fixed to not return zero values in JSON.

https://go.dev/play/p/A2-y61dMkjq